src/main/client/scene/star-field-sprite.ts

```diff
@@ -0,0 +1,35 @@
+import * as Phaser from 'phaser';
+
+interface StarFieldOptions {
+   readonly scene: Phaser.Scene;
+}
+
+export class StarFieldSprite extends Phaser.GameObjects.Sprite {
+   private starFieldTexture;
+
+   constructor(options: StarFieldOptions) {
+      const canvas = document.createElement('canvas') as HTMLCanvasElement;
+      canvas.width = options.scene.game.scale.width * 2;
+      canvas.height = options.scene.game.scale.height * 2;
+
+      const texture = options.scene.textures.addCanvas('star-field', canvas);
+      super(options.scene, options.scene.game.scale.width / 2, options.scene.game.scale.height / 2, 'star-field');
+
+      for (let i = 0; i < 2000; i++) {
+         const radius = Math.random() * 2 + 0.2;
+         const x = Math.floor(Math.random() * canvas.width) + 1;
+         const y = Math.floor(Math.random() * canvas.height) + 1;
```

Comment:
Is this a random between min and max? We could extract this to a util. I think I wrote something like this for previous projects:
`    static randomBetween(min: number, max: number): number {
        return Math.random() * (max - min) + min;
    }`

Reply:
yeah good idea